training/index.md

```diff
@@ -111,6 +111,12 @@ Individual links to course provide more details on the content and prerequisites
               {% else %}
             <a href="{{ course.registration_url }}">Join waiting list</a>
               {% endif %}
+            {% elsif course.registration_status == "closed" %}
+              {% if course.prace_course %}
+            <a href="{{ course.registration_url }}"><img src="img/prace_25.jpg" alt="PRACE"/> Registration closed</a>
+              {% else %}
+            <a href="{{ course.registration_url }}">Registration closed</a>
+              {% endif %}
```

Comment:
@clairbarrass Just wanted to check that you intended to leave live registration links in for closed courses. I can imagine that you may want to for PRACE courses where the Indico page it takes you to will show closed but for non-PRACE courses is just text without a link more appropriate?

If I am mistaken and the links still need to be in, then this PR is fine and I would be happy to approve.

Reply:
Good point.  Have removed the link code around the "Registration closed" for non-PRACE courses.